cluster/service.go

```diff
@@ -172,11 +173,3 @@ func (s *clusterService) refreshCache(ctx context.Context) error {
 	log.Debug(ctx, nil, "write lock released")
```

Comment:
Always unlock in defer right after locking, so if there is panic you won't block the cache forever.

Reply:
done in 2252176